packages/twenty-server/src/engine/api/graphql/workspace-query-runner/workspace-query-runner.service.ts

```diff
@@ -432,6 +433,12 @@ export class WorkspaceQueryRunnerService {
       workspaceId,
       objectMetadataItem,
     );
+
+    const deletedBlocklistItem = await this.handleDeleteBlocklistItem(
```

Comment:
Ideally we would like this in the deleteMany for API users but since this is a temporary implementation I guess it's fine!